### DISCUSSION TITLE
Remove an unassigned face, without removing the picture

### DISCUSSION THREAD

**Discussion by vbelloir** [nextcloud/recognize#1086] [Q&A]
Hi folks, 
I'm using recognize with memories as UI
Before definitely adoping it, I'm playing this a test collection, with very old scanned pictures.  
There is a lot of unassigned face, which is quite normal with only one occurrence.
I don't want to create a cluster with that face, I only want recognize to forget it, without removing the picture.

How can I achieve that?

Thanks.
Vincent

**Comment by carminou** [comment]
Same question

**Comment by marcelklehr** [comment]
This is currently not possible
